core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/spreadsheet/PoolSpreadSheets.java

```diff
@@ -0,0 +1,37 @@
+package uk.ac.bbsrc.tgac.miso.core.data.spreadsheet;
+
+import java.util.Arrays;
+import java.util.List;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Pool;
+
+public enum PoolSpreadSheets implements Spreadsheet<Pool> {
+  TRACKING_LIST("qPCR Tracking List", //
+      Column.forString("Name", Pool::getName), //
+      Column.forString("Alias", Pool::getAlias), //
+      Column.forString("Barcode", Pool::getIdentificationBarcode), //
+      Column.forString("Latest qPCR QC", pool -> pool.getQCs().stream().filter(qc -> qc.getType().getName().equals("qPCR"))
+          .max((qc1, qc2) -> {
```

Comment:
For this, you can use `Comparator.comparing(PoolQC::getDate)` instead of something custom.

Reply:
The reason why I used something custom was because `Comparator.comparing(PoolQC::getDate)` gives the older QC if two qPCRs have the same date. I'm not sure if that is something worth considering.

Reply:
Oh, you can `Comparator.comparing(PoolQC::getDate).reversed()` in that case.

Reply:
That doesn't give the desired outcome. The comparator needs to choose the most recent date, but in the event that two QCs have the same date, it needs to choose the second QC, as it would have been created more recently than the previous one due to the way QCs are stored. This isn't ideal however, as if the first QC was edited after the second QC was created, the second QC will still be chosen as the more recent one. I'm not really sure what the best way to go about deciding between two QCs made on the same date would be.